base.rb

```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
```

Comment:
__[method_definition_validator]__
* Not enough arguments. Did you forget the following arguments? [c] at [base.rb#L15](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/base.rb#L15)
* Not enough arguments. Did you forget the following arguments? [c] at [use.rb#L2](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/use.rb#L2)